common/src/jni/main/cpp/org_conscrypt_NativeCrypto.cpp

```diff
@@ -3898,14 +3897,14 @@ static void NativeCrypto_X509_verify(JNIEnv* env, jclass, jlong x509Ref, jobject
     EVP_PKEY* pkey = fromContextObject<EVP_PKEY>(env, pkeyRef);
     JNI_TRACE("X509_verify(%p, %p)", x509, pkey);
 
-    if (x509 == nullptr) {
-        Errors::jniThrowNullPointerException(env, "x509 == null");
-        JNI_TRACE("X509_verify(%p, %p) => x509 == null", x509, pkey);
+    if (pkey == nullptr) {
```

Comment:
If `fromContextObject` call gets a `nullptr` for `pkey` it will push a `NullPointerException` and at that point the JNI exception will be pending. If `x509` is also `nullptr` the previous line 3902 will try to throw another exception when one is already pending which is a JNI error.

Normally this won't be a problem on production builds, but it aborts in debug Android builds or when checkJni is enabled.